src/util/postgres_client.js

```diff
@@ -1412,19 +1415,25 @@ class PostgresClient extends EventEmitter {
         this.sequences = [];
         const postgres_port = parseInt(process.env.POSTGRES_PORT || '5432', 10);
 
-        // TODO: This need to move to another function
-        this.new_pool_params = {
-
-            // TODO: check the effect of max clients. default is 10
-            max: config.POSTGRES_MAX_CLIENTS,
-
-            host: process.env.POSTGRES_HOST || 'localhost',
-            user: process.env.POSTGRES_USER || 'postgres',
-            password: process.env.POSTGRES_PASSWORD || 'noobaa',
-            database: process.env.POSTGRES_DBNAME || 'nbcore',
-            port: postgres_port,
-            ...params,
-        };
+        if (process.env.POSTGRES_CONNETION_STRING) {
+            /** @type {import('pg').PoolConfig} */
+            this.new_pool_params = {
+                connectionString: process.env.POSTGRES_CONNETION_STRING,
+                ...params,
+            };
+        } else {
+            // TODO: This need to move to another function
+            this.new_pool_params = {
+                host: process.env.POSTGRES_HOST || '127.0.0.1',
+                user: process.env.POSTGRES_USER || 'postgres',
+                password: process.env.POSTGRES_PASSWORD || 'noobaa',
+                database: process.env.POSTGRES_DBNAME || 'nbcore',
```

Comment:
should we move all process.env into the config?

Reply:
decided to leave the old format for now